samples/observability.js

```diff
@@ -0,0 +1,149 @@
+const {BatchSpanProcessor} = require('@opentelemetry/sdk-trace-base');
+const {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMRESATTRS_SERVICE_VERSION,
+  // eslint-disable-next-line n/no-extraneous-require
+} = require('@opentelemetry/semantic-conventions');
+
+const resource = Resource.default().merge(
+  new Resource({
+    [SEMRESATTRS_SERVICE_NAME]: 'spanner-sample',
+    [SEMRESATTRS_SERVICE_VERSION]: 'v1.0.0', // The version of your app running.,
+  })
+);
+
+/*
+ * Uncomment these lines to debug OpenTelemetry.
+const {diag, DiagConsoleLogger, DiagLogLevel} = require('@opentelemetry/api');
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);
+*/
+
+// Create the Google Cloud Trace exporter for OpenTelemetry.
+const {
+  TraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+const exporter = new TraceExporter();
+
+// Optionally, you can enable gRPC instrumentation, by removing this guard.
+if (process.env.SPANNER_ENABLE_GRPC_INSTRUMENTATION === 'true') {
+  // eslint-disable-next-line n/no-extraneous-require
+  const {registerInstrumentations} = require('@opentelemetry/instrumentation');
+  const {GrpcInstrumentation} = require('@opentelemetry/instrumentation-grpc');
+  registerInstrumentations({
+    instrumentations: [new GrpcInstrumentation()],
+  });
+}
+
+function traceAndExportSpans(instanceId, databaseId, projectId) {
+  // Create the OpenTelemetry tracerProvider that the exporter shall be attached to.
+  const provider = new NodeTracerProvider({
+    // Trace every single request to ensure that we generate
+    // enough traffic for proper examination of traces.
+    sampler: new TraceIdRatioBasedSampler(1.0),
+    resource: resource,
+  });
+  provider.addSpanProcessor(new BatchSpanProcessor(exporter));
+
+  // Uncomment this line to make this a global tracerProvider instead of
+  // passing it into SpannerOptions.observabilityConfig.
+  // provider.register();
+
+  // Acquire the tracer.
+  const tracer = provider.getTracer('MyApp');
+
+  // Start our user defined trace.
+  tracer.startActiveSpan('SpannerTracingQuickstart', async span => {
+    // Create the Cloud Spanner Client.
+    const {Spanner} = require('@google-cloud/spanner');
+    const spanner = new Spanner({
+      projectId: projectId,
+      observabilityOptions: {
+        // Optional, but you could rather register the global tracerProvider.
+        tracerProvider: provider,
+        // This option can also be enabled by setting the environment
+        // variable `SPANNER_ENABLE_EXTENDED_TRACING=true`.
+        enableExtendedTracing: true,
+      },
+    });
+
+    // Acquire the database handle.
+    const instance = spanner.instance(instanceId);
+    const database = instance.database(databaseId);
+
+    try {
+      const query = {
+        sql: 'SELECT 1',
+      };
+      const [rows] = await database.run(query);
+
+      for (const row of rows) {
+        console.log(`Result: ${row.toString()}`);
```

Comment:
@odeke-em  it is printing the response as [object, object].

maybe we can print the actual response we are getting here

Reply:
@alkatrivedi gotcha, it is now just invoke `console.log(row)`